vm/actor_interface/src/adt/map.rs

```diff
@@ -63,7 +64,7 @@ where
     }
 
     /// Returns a reference to the value corresponding to the key.
-    pub fn get<Q: ?Sized>(&self, k: &Q) -> Result<Option<&V>, AnyhowError>
+    pub fn get<Q: ?Sized + Hash>(&self, k: &Q) -> Result<Option<&V>, AnyhowError>
```

Comment:
Do we need `Hash` here? If yes, why?

```suggestion
    pub fn get<Q: ?Sized>(&self, k: &Q) -> Result<Option<&V>, AnyhowError>
```